app/registries/foreman/access_permissions.rb

```diff
@@ -721,6 +721,11 @@
       :"api/v2/personal_access_tokens" => [:destroy]
   end
 
+  permission_set.security_block :json_web_tokens do |map|
```

Comment:
do we need to distinguish json_web_tokens vs personal_access_tokens? it seems fairly similar to me and I'd like to limit count of permissions we have if possible.
Or actually isn't it enough when I can manage the user to manage their JWT ?

Reply:
I don't have any strong opinions about it, I'm fine with having only `:edit_users` permission for JWT
But during the reviews I updated permissions logic at several times, and I really don't wan to edit it again just to have it discarded by another new idea in follow up review.

I'm for the idea to have only `:edit_users` permission for `JWT` actions, let's see what community thinks about it

Reply:
I'd agree. I'd even go further and say we should consider dropping the personal access token permissions as well. The fact these are internally stored as separate models shouldn't matter to the user. 
These should be similar to other user settings and preferences, with a little bit of difference:
- A user has full control of their own (CRUD) and permissions don't even need to be checked.
- A user with permission to `edit_users` (e.g. admin, manager, system admin roles) can delete/revoke other users' tokens. 
- Since these are security-sensitive, I'd suggest preventing such roles from viewing and creating them. If really needed it can be done by impersonating the user, in which case it is also recorded in the logs.
- Anyone else shouldn't be allowed to do anything about other users' tokens.